package.json

```diff
@@ -28,7 +28,6 @@
   "pnpm": {
     "patchedDependencies": {
       "postcss-css-variables@0.19.0": "patches/postcss-css-variables@0.19.0.patch",
-      "process@0.11.10": "patches/process@0.11.10.patch",
```

Comment:
This package was one that we used as a polyfill that isn't needed and isn't required to be patched anymore!